cider/exec/nextgen/parsers/RelAlgEUParser.cpp

```diff
@@ -138,11 +135,28 @@ OpPipeline toOpPipeline(const RelAlgExecutionUnit& eu) {
   ops.emplace_back(createOpNode<operators::FilterNode>(filters));
 
   ExprPtrVector projs;
+  ExprPtrVector aggs;
+  ExprPtrVector groupbys;
+
   for (auto& targets_expr : eu.shared_target_exprs) {
-    projs.push_back(targets_expr);
+    if (targets_expr->get_contains_agg()) {
+      aggs.push_back(targets_expr);
+    } else {
+      projs.push_back(targets_expr);
+    }
```

Comment:
how do we keep the order of the output column in `select a, sum(*), b, count(*) .. group by a, b`

Reply:
this will fix in pr which supports groupby